examples/darwin-framework-tool/commands/common/CHIPToolKeypair.h

```diff
@@ -17,15 +17,15 @@
  */
 
 #import <Matter/Matter.h>
-#include <crypto/CHIPCryptoPAL.h>
 
-@interface CHIPToolKeypair : NSObject <MTRKeypair>
-- (BOOL)initialize;
-- (NSData *)signMessageECDSA_RAW:(NSData *)message;
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Keypair : NSObject <MTRKeypair>
```

Comment:
since we're in objc-land here, we need to prefix our class names to avoid collisions.  if it's `darwin-framework-tool`-specific, maybe `DFTKeypair`?

Reply:
(this probably isn't as important as it is if it's actual framework code but it's probably a good habit for us to exercise everywhere)